apps/report-documents-not-received/sections/summary-data-sections.js

```diff
@@ -18,6 +18,30 @@ module.exports = {
       step: '/documents-not-received-main-applicant',
       field: 'dnr-nationality'
     },
+    {
+      step: '/documents-not-received-reference-number',
+      field: 'dnr-reference-number'
+    },
+    {
+      step: '/documents-not-received-reference-number',
+      field: 'dnr-reference-number',
+      parse: (value, req) => {
```

Comment:
I think the label should be as per the selected item from the reference page so we may need to have separate entries for each field otherwise this will always say `Reference number` where it can be `Record number,` `Case ID` and so on

Reply:
Looks okay